### ISSUE TITLE
UGA's contributing org isn't matching up with their records

### ISSUE THREAD

**Issue by sroosa** [WGBH-MLA/AAPB2#1849]
There is an organization page for The Walter J. Brown Media Archives & Peabody Awards Collection at the University of Georgia (https://americanarchive.org/participating-orgs/NCSG55097). 

There are also some records that SHOULD have UGA as a contributing organization (based on the organization annotation in  the instantiation) like this one: https://americanarchive.org/catalog/cpb-aacip_526-n29p26r78s.pbcore. But it seems like that data isn't getting mapped into the contributing organization fields when the xml is ingest since The View All Records link on their org page doesn't link to any records and their name doesn't show up in the facet.

Is it because of the ampersand?

**Comment by jasoncorum** [comment]
@sroosa - i think we need to handle this more like we do the Stanford Hoover library stuff. if you look at this pbcore record:

https://americanarchive.org/catalog/cpb-aacip_514-s17sn0215b.pbcore

we use just "Stanford University" for the organization:

`<pbcoreAnnotation annotationType="organization">Stanford University</pbcoreAnnotation>`

And then we use the full Hoover name as the "Short name" on the organization page:

```
## Short name

Hoover Institution Library &amp; Archives, Stanford University
```

**Comment by sroosa** [comment]
We decided we want to address this in the ingest, so that if PBCore comes in with the ampersand in the Org name, it maps it to something without that.

**Comment by sroosa** [comment]
Closed by #1887
